qiskit/aqua/operators/converters/abelian_grouper.py

```diff
@@ -88,30 +87,33 @@ def group_subops(cls, list_op: ListOp, fast: bool = True, use_nx: bool = False)
 
         Args:
             list_op: The Operator to group into Abelian groups
-            fast: Enable the fast commutation graph generation if all operators are Pauli operators
-            use_nx: Enable networkx.coloring.greedy_color instead of the numpy-based coloring
+            fast: DEPRECATED. Enable the fast commutation graph generation if all operators are
+                Pauli operators
+            use_nx: DEPRECATED. Enable networkx.coloring.greedy_color instead of the numpy-based
+                coloring
 
         Returns:
             The grouped Operator.
 
         Raises:
             AquaError: Any of list_op's sub-ops do not have a ``commutes`` method.
         """
-        if any(not hasattr(op, 'commutes') for op in list_op.oplist):
-            raise AquaError('Cannot determine Abelian groups if an Operator in list_op does not '
-                            'contain a `commutes` method'.format())
-
-        if fast and all(isinstance(op, PauliOp) for op in list_op.oplist):
-            edges = cls._commutation_graph_fast(list_op)
-        else:
-            edges = cls._commutation_graph(list_op)
+        if not fast or use_nx:
+            warnings.warn('Options `fast` and `use_nx` of `AbelianGrouper.group_subops` are '
+                          'deprecated as of 0.7.5 and will be removed no sooner than 3 months '
+                          'after the release.')
+
+        for op in list_op.oplist:
+            if not isinstance(op, PauliOp):
```

Comment:
The test check `not isinstance PauliOp` and the error message seem to mismatch - the message talks about a `commutes` method, which is not checked for in the test, rather it just checks if it's a PauliOp. Should the check be looking for a method commute on the operators in the list?, or should the message be changed to say this only works on PauliOps?

Reply:
You are right. I fixed it.